src/features/offer/components/OfferTitle/OfferTitle.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import { accessibilityAndTestId } from 'libs/accessibilityAndTestId'
+import { Typo } from 'ui/theme'
+import { getHeadingAttrs } from 'ui/theme/typographyAttrs/getHeadingAttrs'
+
+interface Props {
+  offerName: string
+}
+
+export function OfferTitle({ offerName }: Readonly<Props>) {
+  const titleProps = {
+    adjustsFontSizeToFit: true,
+    allowFontScaling: false,
+    ...accessibilityAndTestId(`Nom de l’offre\u00a0: ${offerName}`),
+    ...getHeadingAttrs(1),
+  }
+
+  return <Typo.Title3 {...titleProps}>{offerName}</Typo.Title3>
+}
```

Comment:
Ta variable `titleProps` pose 2 problèmes :
- Elle n'est pas memoized donc chaque render de `OfferTitle` (qui n'est pas memoized non plus) va rerender le composant
- Elle n'est pas utile et va à l'encontre de la façon dont on appelle un composant

```suggestion
export function OfferTitle({ offerName }: Readonly<Props>) {
  return (
    <Typo.Title3
      adjustsFontSizeToFit
      allowFontScaling={false}
      {...accessibilityAndTestId(`Nom de l’offre\u00a0: ${offerName}`)}
      {...getHeadingAttrs(1)}
    >
      {offerName}
    </Typo.Title3>
}
```